scripts/ci/getChangedFiles.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+keg_get_changed_files(){
+  # Pull down origin, so we have access to all branches
+  git fetch origin --force --depth=1
+
+  # Get the current branch to check if we are on develop
+  local CURRENT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+  local CHECK_BRANCH="develop"
+
+  # If we are on the develop branch, then check file changes against the master branch
+  if [[ "$CURRENT_BRANCH" == "develop" ]]; then
```

Comment:
on the `build` workflow, im running into an issue where it's not actually building any of the repos.

this commit should have some build file updates for jsutils https://github.com/daniel112/keg-hub/commit/083e5bad43abbc7bbc214e5cca622b91a3799245

here are the file changes when I run `yarn build` on jsutils locally
- [image](https://user-images.githubusercontent.com/3317835/103705518-671bcb80-4f68-11eb-8d76-3db74e1f828f.png)

if I understand this right, when it's on the `master` branch, it's comparing itself to `develop`, so it wont actually find any file changes

Reply:
This should be resolved now that the build action runs on pushes to develop